### PR TITLE
feat: add custom issue forms for Finder

### LINKED ISSUE #250: feat: add custom issue forms for Finder
### Description

Add custom issue forms for the Finder project (From .github), including: 

- General Feature Request 
- Documentation issue
- Bug 
- Other
- 🙋🏾🙋🏼‍Question

### Screenshots

>**Note**
> Use `Finder` in instead of `LinkFree`.

image

### Additional information

All file should be added in a new directory `"ISSUE_TEMPLATE"` in `./.github`

### PR DESCRIPTION
**Pull request by kumarsonsoff3**
## Fixes Issue
closes #250 

## Changes proposed

Added custom forms for the Finder -
- General Feature Request
- Documentation issue
- Bug
- Other
- 🙋🏾🙋🏼‍Question

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/good-first-issue-finder/pull/251"></a>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/other.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      label: Additional information
+      description: Is there anything else we should know about this issue?
+    validations:
+      required: false
+  - type: markdown
+    attributes:
+      value: |
+        You can also join our Discord community [here](http://discord.eddiehub.org)
+        Feel free to check out other cool repositories of the EddieHub Community [here](https://github.com/EddieHubCommunity)
+
```
Comment (eddiejaoude) [inline .github/ISSUE_TEMPLATE/other.yml:28]: GitHub recommends that all files need a blank line to end the file (no space)

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name: 💡 General Feature Request
+description: Have a new idea/feature for Finder? Please suggest!
+title: "[FEATURE] <description>"
+labels: ["enhancement"]
```
[the remark this answers is not in the document]
Comment (kumarsonsoff3) [reply .github/ISSUE_TEMPLATE/feature_request.yml:4]: We have `"✨goal: improvement"`, maybe we can used that here!?

**Review by Cahllagerfeld — APPROVED** [review]
Awesome, thank you 🚀

**Pull request merged**